tests/console/system_prepare.pm

```diff
@@ -75,7 +76,7 @@ sub run {
     }
 
     # Save output info to logfile
-    if (is_sle) {
+    if (is_sle && get_required_var('FLAVOR') =~ /Migration/) {
```

Comment:
This condition will change the flow in comparison how it was before. What's about adding it to the line where we call `services::registered_addons::full_registered_check`?

Reply:
As all of content which 'if (is_sle)' has, was added by me. I want those part can be used only for our team. As maybe other team will not use it. So I add 'get_required_var('FLAVOR') =~ /Migration/' to 'if ' condition.

Reply:
Alright, with this condition we can merge, as it won't affect clean installations. Thanks!